lib/widgets/content.dart

```diff
@@ -32,33 +32,72 @@ import 'text.dart';
+      colorDirectMentionBackground: const HSLColor.fromAHSL(0.2, 240, 0.7, 0.7).toColor(),
+      colorGlobalTimeBackground: const HSLColor.fromAHSL(1, 0, 0, 0.93).toColor(),
+      colorGlobalTimeBorder: const HSLColor.fromAHSL(1, 0, 0, 0.8).toColor(),
+      colorMathBlockBorder: const HSLColor.fromAHSL(0.15, 240, 0.8, 0.5).toColor(),
+      colorMessageMediaContainerBackground: const Color.fromRGBO(0, 0, 0, 0.03),
+      colorThematicBreak: const HSLColor.fromAHSL(1, 0, 0, .87).toColor(),
+      textStylePlainParagraph: _plainParagraphCommon(context)
+        .apply(color: const HSLColor.fromAHSL(1, 0, 0, 0.15).toColor())
+        .copyWith(debugLabel: 'ContentTheme.textStylePlainParagraph'),
+      codeBlockTextStyles: CodeBlockTextStyles.light(context),
+      textStyleError: const TextStyle(fontSize: kBaseFontSize, color: Colors.red)
+        .merge(weightVariableTextStyle(context, wght: 700)),
+      textStyleErrorCode: kMonospaceTextStyle
+        .merge(const TextStyle(fontSize: kBaseFontSize, color: Colors.red)),
+      textStyleInlineCode: kMonospaceTextStyle.merge(TextStyle(
+        backgroundColor: const HSLColor.fromAHSL(0.06, 0, 0, 0).toColor())),
+      textStyleInlineMath: kMonospaceTextStyle.merge(TextStyle(
+        // TODO(#46) this won't be needed
+        backgroundColor: const HSLColor.fromAHSL(1, 240, 0.4, 0.93).toColor())),
+    );
+  }
 
-      color: const HSLColor.fromAHSL(1, 0, 0, 0.15).toColor(),
-      fontSize: kBaseFontSize,
-      letterSpacing: 0,
-      textBaseline: localizedTextBaseline(context),
-      height: (22 / kBaseFontSize),
-      leadingDistribution: TextLeadingDistribution.even,
-      decoration: TextDecoration.none,
-      fontFamily: kDefaultFontFamily,
-      fontFamilyFallback: defaultFontFamilyFallback,
-    )
-      .merge(weightVariableTextStyle(context))
-      .copyWith(debugLabel: 'ContentTheme.textStylePlainParagraph'),
-    codeBlockTextStyles = CodeBlockTextStyles.light(context),
-    textStyleError = const TextStyle(fontSize: kBaseFontSize, color: Colors.red)
-      .merge(weightVariableTextStyle(context, wght: 700)),
-    textStyleErrorCode = kMonospaceTextStyle
-      .merge(const TextStyle(fontSize: kBaseFontSize, color: Colors.red));
+  factory ContentTheme.dark(BuildContext context) {
+    return ContentTheme._(
+      colorCodeBlockBackground: const HSLColor.fromAHSL(0.04, 0, 0, 1).toColor(),
+      colorDirectMentionBackground: const HSLColor.fromAHSL(0.25, 240, 0.52, 0.6).toColor(),
+      colorGlobalTimeBackground: const HSLColor.fromAHSL(0.2, 0, 0, 0).toColor(),
+      colorGlobalTimeBorder: const HSLColor.fromAHSL(0.4, 0, 0, 0).toColor(),
+      colorMathBlockBorder: const HSLColor.fromAHSL(1, 240, 0.4, 0.4).toColor(),
+      colorMessageMediaContainerBackground:
+        const HSLColor.fromAHSL(0.03, 0, 0, 1).toColor(),
+      colorThematicBreak:
+        const HSLColor.fromAHSL(1, 0, 0, .87).toColor().withOpacity(0.2),
```

Comment:
nit:
```suggestion
      colorMessageMediaContainerBackground: const HSLColor.fromAHSL(0.03, 0, 0, 1).toColor(),
      colorThematicBreak: const HSLColor.fromAHSL(1, 0, 0, .87).toColor().withOpacity(0.2),
```

to keep it parallel with the light constructor, for comparison.

(It's a little regrettable that the 0.2 opacity gets pushed off past 80 columns that way. But better that than breaking parallelism.)